idl/chimera.pro

```diff
@@ -154,7 +168,7 @@ dat1=data[*,*,1]
 dat2=data[*,*,2]
 
 ;======Get pixels with useful intensities and on disk======
-r = ind[0].r_sun
+r = rs
```

Comment:
Is this reassignment necessary? Isn't better to change all the `r` by `rs`? or better, by `r_sun`?